examples/react/multi-file-upload/src/ContentPage.jsx

```diff
@@ -33,46 +33,46 @@ export function ContentPage () {
+    return <Uploading files={files} storedDAGShards={storedDAGShards} />
   }
 
   if (status === 'done') {
-    return error ? <Errored error={error} /> : <Done files={files} dataCid={dataCid} uploadedCarChunks={uploadedCarChunks} />
+    return error ? <Errored error={error} /> : <Done files={files} dataCid={dataCid} storedDAGShards={storedDAGShards} />
   }
 
   return (
     <form onSubmit={handleUploadSubmit}>
-      <div className='mb3'>
-        <label htmlFor='files' className='db mb2'>Files:</label>
-        {allowDirectory
-          ? <input id='file' className='db pa2 w-100 ba br2' type='file' webkitdirectory='true' onChange={e => setFiles(Array.from(e.target.files))} required />
-          : <input id='file' className='db pa2 w-100 ba br2' type='file' multiple onChange={e => setFiles(Array.from(e.target.files))} required />}
-      </div>
-      <div className='mb3'>
-        <label>
-          <input type='checkbox' value={allowDirectory} onChange={e => setAllowDirectory(e.target.checked)} /> Allow directory selection
-        </label>
-      </div>
-      {files.length === 1
-        ? (
-          <div className='mb3'>
-            <label>
-              <input type='checkbox' value={wrapInDirectory} onChange={e => setWrapInDirectory(e.target.checked)} /> Wrap file in a directory
-            </label>
-          </div>
-          )
-        : null}
-      <button type='submit' className='ph3 pv2'>Upload</button>
-    </form>
+    <div className='mb3'>
+      <label htmlFor='files' className='db mb2'>Files:</label>
+      {allowDirectory
+        ? <input id='file' className='db pa2 w-100 ba br2' type='file' webkitdirectory='true' onChange={e => setFiles(Array.from(e.target.files))} required />
+        : <input id='file' className='db pa2 w-100 ba br2' type='file' multiple onChange={e => setFiles(Array.from(e.target.files))} required />}
+    </div>
+    <div className='mb3'>
+      <label>
+        <input type='checkbox' value={allowDirectory} onChange={e => setAllowDirectory(e.target.checked)} /> Allow directory selection
+      </label>
+    </div>
+    {files.length === 1
+      ? (
+        <div className='mb3'>
+          <label>
+            <input type='checkbox' value={wrapInDirectory} onChange={e => setWrapInDirectory(e.target.checked)} /> Wrap file in a directory
+          </label>
+        </div>
+        )
+      : null}
+    <button type='submit' className='ph3 pv2'>Upload</button>
+  </form>
   )
 }
 
-const Uploading = ({ files, dataCid, uploadedCarChunks }) => (
+const Uploading = ({ files, storedDAGShards }) => (
   <div className='flex items-center'>
     <div className='spinner mr3 flex-none' />
     <div className='flex-auto'>
-      <p className='truncate'>Uploading DAG for {files.length > 1 ? `${files.length} files` : files[0].name}</p>
-      {uploadedCarChunks.map(({ cid, size }) => (
+    <p className='truncate'>Uploading DAG for {files.length > 1 ? `${files.length} files` : files[0].name}</p>
```

Comment:
looks like it should be nested one indent more here
```suggestion
      <p className='truncate'>Uploading DAG for {files.length > 1 ? `${files.length} files` : files[0].name}</p>
```